docs/guis/developing/ide_install.md

```diff
@@ -5,7 +5,7 @@ There is also an [installation guide available][accsoft_eclipse_wiki]{target=_bl
 
 For people using pycharm it might make sense to use IntelliJ IDEA, however we do not yet know how to export a JAR (see also [GUI Releases](releases.md)).
 
-As both IDEs require [CommonBuildNextGeneration (CBNG)][cbng_wiki]{target=_blank} to resolve dependencies and make releases, one should either run these from somewhere in the Technical Network (e.g. from the `dev-server`) or mount the required paths via `sshfs` as described [here](../../resources/howto/setup.md#mounting-tn-resources-on-gn-machines).
+As both IDEs require [CommonBuildNextGeneration (CBNG)][cbng_wiki]{target=_blank} to resolve dependencies and make releases, one should either run these from somewhere in the Technical Network (e.g. from the `dev-server`) or [mount the required paths via `sshfs`][mount_tn].
```

Comment:
This should be [mounting_tn]{target=_blank}

Reply:
I thought new tabs only for external links?

Reply:
You're right!